### ISSUE TITLE
Item Reorder Gesture problem in RTL

### ISSUE THREAD

**Issue by imanolarellano** [ionic-team/ionic#12527]
**Ionic version:**
[ ] **1.x** (For Ionic 1.x issues, please use https://github.com/ionic-team/ionic-v1)
[x] **2.x**
[ ] **3.x**

**I'm submitting a ...**
[x] bug report
[ ] feature request

**Current behavior:**
When you try to do drag and drop to reorder a list in RTL the list cannot reorder. 

**Expected behavior:**
You can to drag and drop on list.

**Steps to reproduce:**
Try to reorder a list in RTL.
https://ionicframework.com/docs/api/components/item/ItemReorder/

**Ionic info:**

```
Cordova CLI: 6.5.0 
Ionic Framework Version: 2.0.1
Ionic CLI Version: 2.2.1
Ionic App Lib Version: 2.2.0
Ionic App Scripts Version: 1.0.0
ios-deploy version: 1.9.1 
ios-sim version: Not installed
OS: macOS Sierra
Node Version: v6.9.4
Xcode version: Xcode 8.3.3 Build version 8E3004b
```

**Comment by jgw96** [comment]
Thanks for opening an issue with us. @AmitMY would you mind taking a look at this?

**Comment by AmitMY** [comment]
Sure.
The thing is, you are using v2.x. We fixed RTL in version 3.4, so I recommend you upgrade to latest (it is really simple, and there are guides online like [this one](https://www.youtube.com/watch?v=oQJMUOznMrA) - I recommend first upgrading to 3.0.0, and then to 3.6.0 (will be easier)
* Note that it is really recommended to have a version >=3.4 if you are using RTL, as there are plenty of stuff that won't work, and you can't fix just from scss.

Another thing, is that you can't control RTL from the browser. meaning, `dir="rtl"` is plenty, but is not enough. You also need to do `this.platform.setDir('rtl')`

Full explanation on RTL is [here](http://blog.ionic.io/ionic-and-rtl/)
